src/palette/view/widgets/palettewidget.cpp

```diff
@@ -821,6 +821,9 @@ void PaletteWidget::dragEnterEvent(QDragEnterEvent* event)
                 || suffix == "jpg"
                 || suffix == "jpeg"
                 || suffix == "png"
+                //|| suffix == "bmp"
+                //|| suffix == "tif"
+                //|| suffix == "tiff"
```

Comment:
Doesn't seem to be needed, but I'd rather understand why

Reply:
Hypothesis: this code just determines whether the UI will show a "copy here" symbol next to the mouse cursor (or whatever each OS does when you drag something somewhere where it can be dropped). 
Then, in `PaletteWidget::dropEvent`, we just allow whatever file to be dropped, not checking its filename extension, regardless of whether `PaletteWidget::dragEnterEvent` told the OS to indicate that the drop would be accepted or rejected.

No idea whether that's anywhere near true, but that's just a guess.

Reply:
Well, that was my suspicion too, but it doesn't seem to be the case. a "copy here" does get shown on any case, regardless of he exteion, but actually doesn't works at all when draging from outsdide MuseScore and does regardless of extentsion when done from a score.
That's what's confusing me...

Reply:
For now I think it's better to just uncomment these lines, because having them commented looks even more confusing. You could add a TODO comment though, saying that these lines don't seem to do the expected thing.

Reply:
done